data/resources/ui/about.ui.in

```diff
@@ -3,14 +3,19 @@
   <object class="AdwAboutWindow" id="about">
     <property name="application-icon">icon</property>
     <property name="application-name">Tubefeeder</property>
-    <property name="comments">A Youtube, Lbry and Peertube client made for the Pinephone</property>
-    <property name="copyright">© 2022 Julian Schmidhuber</property>
+    <property name="copyright">Copyright © 2022 Julian Schmidhuber</property>
     <property name="developer-name">Julian Schmidhuber</property>
+    <property name="developers">Julian Schmidhuber</property>
     <property name="issue-url">https://github.com/Tubefeeder/Tubefeeder/issues</property>
     <property name="license-type">GTK_LICENSE_GPL_3_0</property>
-    <property name="support-url">https://matrix.to/#/%23tubefeeder:matrix.org</property>
-    <property name="translator-credits" translatable="yes">translators</property>
+    <property name="support-url">https://matrix.to/#/#tubefeeder:matrix.org</property>
+    <property name="translator-credits" translatable="yes">David Lapshin &lt;ddaudix@gmail.com&gt;
+      vikdevelop &lt;super-vik1@protonmail.com&gt;
+      Julian Schmidhuber &lt;internationalization@schmiddi.anonaddy.com&gt;
+      Michael Moroni &lt;michaelmoroni@disroot.org&gt;
+      Heimen Stoffels &lt;vistausss@fastmail.com&gt;</property>
```

Comment:
```suggestion
    <property name="translator-credits" translatable="yes">translator-credits</property>
```

Maybe we miss-understand each other. That is what I mean.

Reply:
Hmm... I'm guessing this has to be done in the source code and not in the UI spec. The UI spec takes it literally so it will just show `translator-credits` in the translator credits, as can be seen in the current spec:

![image](https://user-images.githubusercontent.com/50847364/194080850-167fa3e7-e5e5-4cb3-be99-786793efe1df.png)